deployments/salt/pillar.example

```diff
@@ -0,0 +1,19 @@
+isignalfx-agent:
+  repo_base_url: https://dl.signalfx.com
+  package_stage: beta
```

Comment:
I would make this `final` instead of `beta` since most people are going to want to use that.

Reply:
done